src/app/subnets/views/VLANDetails/ConfigureDHCP/ConfigureDHCP.tsx

```diff
@@ -156,7 +164,7 @@ const ConfigureDHCP = ({ closeForm, id }: Props): JSX.Element | null => {
               enableDHCP: true,
               endIP: "",
               gatewayIP: "",
-              primaryRack: vlan.primary_rack || "",
+              primaryRack: vlan.primary_rack || vlan.rack_sids[0] || "",
```

Comment:
I think a comment here about why assigning `vlan.rack_sids[0]` might be necessary.

I'm missing a little context here, but is this just picking any controller to appear as selected in the "Primary rack" select box? It seems a little strange that one of the options can be selected in the field without it setting the value in Formik.

Reply:
Fair point, it might have been a little confusing indeed.

We had a fair bit of complexity here that could lead to more bugs further down the line. I changed the approach and added an empty default option to the field instead which is now compatible with all potential states it can be in (we had no empty - `""` - option on the list previously which caused this bug). The browser selected the only available value which was out of sync with Formik.

This is now consistent with the behaviour of most other forms we have.